NuKeeper.Gitlab/Model/MergeRequest.cs

```diff
@@ -21,5 +21,8 @@ public class MergeRequest
 
         [JsonProperty("remove_source_branch")]
         public bool RemoveSourceBranch { get; set; }
+
+        [JsonProperty("lables")]
```

Comment:
I think this should be `labels`.

Reply:
Haha, damn, I was quite sure to copy it. Silly, thanks for taking a look, Max!

Reply:
No problem, that is why we do code reviews :)

Reply:
And that's fixed... 🤦‍♂